mappings/net/minecraft/container/Container.mapping

```diff
@@ -29,6 +29,11 @@ CLASS asw net/minecraft/container/Container
 		ARG 1 stack
 		ARG 2 startIndex
 		ARG 3 endIndex
+	METHOD a onCraftingContentChanged (Lbbp;Larb;Lafi;Latz;)V
```

Comment:
Is this only used for crafting tables still, or also other Recipe-based inventories?

Reply:
It's only used for crafting tables and in-hand crafting.